src/generator.ts

```diff
@@ -338,12 +338,17 @@ function generateStandaloneEnum(ast: TEnum, options: Options): string {
   )
 }
 
+function notUndefined<T>(x: T | undefined): x is T {
+  return x !== undefined
+}
+
 function generateStandaloneInterface(ast: TNamedInterface, options: Options): string {
+  const superTypeNames = ast.superTypes.map(x => x.standaloneName).filter(notUndefined)
```

Comment:
This is needed because `ast.superTypes` now returns `TNamedInterface | TCustomType` and `TCustomType` doesn't override `standaloneName` to be a required property like `TNamedInterface` does.

I don't see `TCustomType` used anywhere except in this feature. I could thus also make `standaloneName` required in `TCustomType` or I could create a named `TCustomType` that way errors can be caught at parse time instead of at generation time

Reply:
Let's see if we can explicitly handle this case upstream (see my previous suggestion), instead of needing to implicitly handle it here.